fury/primitive.py

```diff
@@ -851,3 +852,223 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
         triangles = (np.array(triangles).reshape(2 * sectors, 3))
 
     return vertices, triangles
+
+
+def init_parametric(u_lower_bound, u_upper_bound, v_lower_bound,
+                    v_upper_bound, npoints):
+    u = np.linspace(u_lower_bound, u_upper_bound, npoints)
+    v = np.linspace(v_lower_bound, v_upper_bound, npoints)
+    u, v = np.meshgrid(u, v)
+    u = u.reshape(-1)
+    v = v.reshape(-1)
+    return u, v, np.sin, np.cos, np.mean
+
+
+def prim_para_mobius_strip(npoints=100):
+    """Return vertices and triangle for a Möbius strip
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our Möbius strip
+    triangles: ndarray
+        triangles that compose our Möbius strip
+    """
+
+    u, v, sin, cos, mean = init_parametric(0, 2*np.pi, -1, 1, npoints)
+    points2D = np.vstack([u, v]).T
+    tri = Delaunay(points2D)
+    triangles = tri.simplices
+
+    x = (1 + v/2 * cos(u/2)) * cos(u)
+    y = (1 + v/2 * cos(u/2)) * sin(u)
+    z = v/2 * sin(u/2)
+    # Centering the surface
+    x -= mean(x)
+    y -= mean(y)
+    z -= mean(z)
+    xyz = np.vstack([x, y, z]).T
+    vertices = np.ascontiguousarray(xyz)
+    return vertices, triangles
```

Comment:
duplicated everywhere. we should avoid that. Only `x,y,z` are changing in your function.

you can rename `init_parametric` by `build_parametric` and then, add `x, y, z` as mandatory parameter

Reply:
Done